tests/components/test-components.php

```diff
@@ -883,7 +883,7 @@ public function test_component_post_social_sharing() {
 					],
 					'platformData' => [],
 					'postId'       => $this->get_post_id(),
-					'title'        => get_the_title( $this->get_post_id() ),
+					'title'        => 'It’s like this & that.',
```

Comment:
Do we need to update the `get_the_title()` helper to make encoding optional?

Reply:
Hmm, there isn't an existing helper, we're just using the core function. I can add a helper though.